packages/loader/container-definitions/src/fluidPackage.ts

```diff
@@ -3,82 +3,72 @@
  * Licensed under the MIT License.
  */
 
-/**
- * Person definition in a npm script
- */
-export interface IPerson {
-    name: string;
-    email: string;
-    url: string;
-}
+ /**
+  * Specifies an environment on Fluid property of a IFluidPackage
+  */
+export interface IFluidPackageEnvironment {
 
-/**
- * Typescript interface definition for fields within a npm module's package.json.
- */
-export interface IPackage {
-    // General access for extended fields
-    [key: string]: any;
-    name: string;
-    version: string;
```

Comment:
We currently use this for our "manifest" caching logic in the FFX M365 code loader to know if we are updating the cache to point to a new container bits and preloading the new container code when that happens. We could probably update it to compare the files array instead of comparing the versions, so I think the loss of version will probably be ok

Reply:
you can continue to use version via the general indexer below `[key: string]: any;` or even better craft a concrete type for your code expected package format. this is just defining the minimal format